### PR TITLE
Reporting: Document layouts

### PR DESCRIPTION
**Pull request by sakjur**
**What this PR does / why we need it**:

We should provide sample pictures of what to export from a rendered report, this PR adds that to the reporting documentation (pictures are already in the website repo).

### CODE REVIEW

File: docs/sources/enterprise/reporting.md [hunk 1]
```diff
@@ -77,6 +77,15 @@ Weekly reports are generated once per week. All fields are required.
 
 Reports which are scheduled to never be sent have no parameter and will not be sent to the scheduler. They may be manually generated from the **Send test email** prompt or via the API.
 
+### Layout
+
+> We're actively working on developing new report layout options. [Contact us](https://grafana.com/contact?about=grafana-enterprise) if you would like to get involved in the design process.
+
+Name | Introduced | Description | Preview
```
Comment (AgnesToulet) [inline docs/sources/enterprise/reporting.md:84]:
```suggestion
Name | Support | Description | Preview
```

**Review by AgnesToulet — COMMENTED** [review]
I think it would be best to put this part above "Scheduling" as it's more important and it also follows the form order.
Otherwise, LGTM, simple but clear! Although, you should add SEs in the reviewers to be sure it's what they expect.

File: docs/sources/enterprise/reporting.md [hunk 2]
```diff
@@ -77,6 +77,15 @@ Weekly reports are generated once per week. All fields are required.
 
 Reports which are scheduled to never be sent have no parameter and will not be sent to the scheduler. They may be manually generated from the **Send test email** prompt or via the API.
 
+### Layout
+
+> We're actively working on developing new report layout options. [Contact us](https://grafana.com/contact?about=grafana-enterprise) if you would like to get involved in the design process.
```
Comment (oddlittlebird) [inline docs/sources/enterprise/reporting.md:82]: Should they use keywords or something to help route the message properly?

**Review by oddlittlebird — APPROVED** [review]
One question, looks good otherwise.

**Review by jongyllen — APPROVED** [review]
LGTM

File: docs/sources/enterprise/reporting.md (hunk 2 above)
Comment (sakjur) [reply docs/sources/enterprise/reporting.md]: Good point, I'm working on getting that working for us

File: docs/sources/enterprise/reporting.md [hunk 3]
```diff
@@ -77,6 +77,15 @@ Weekly reports are generated once per week. All fields are required.
 
 Reports which are scheduled to never be sent have no parameter and will not be sent to the scheduler. They may be manually generated from the **Send test email** prompt or via the API.
```
[the remark this answers is not in the document]
Comment (sakjur) [reply docs/sources/enterprise/reporting.md]: Yes, good point. I'll add a task for me to do that  🙂

**Review by xlson — APPROVED** [review]
Looks great! Clean and clear.

File: docs/sources/enterprise/reporting.md (hunk 2 above)
Comment (sakjur) [reply docs/sources/enterprise/reporting.md]: ✅ I've added the parameters to the contact form now, so it should work when both grafana.com and the docs are running with my changes (and it'll work but not with our custom pre-formatted text until then).

**Pull request merged**